web/e2e-tests/mention.test.ts

```diff
@@ -39,4 +39,21 @@ async function test_mention(page: Page): Promise<void> {
     await common.check_messages_sent(page, "zhome", [["Verona > Test mention all", ["@all"]]]);
 }
 
+async function test_change_private_mention_to_silent_mention(page: Page): Promise<void> {
+    await common.log_in(page);
+    await page.click("#left-sidebar-navigation-list .top_left_all_messages");
+    await page.waitForSelector("#zhome .message_row", {visible: true});
+
+    await page.keyboard.press("KeyX");
+    await page.waitForSelector("#private_message_recipient", {visible: true});
+    await common.fill_form(page, "form#send_message_form", {content: "Test private message."});
+    await common.pm_recipient.set(page, "cordelia@zulip.com");
+    await common.select_item_via_typeahead(page, "#compose-textarea", "@**Cord", "Cordelia");
+    await common.ensure_enter_does_not_send(page);
+    await page.waitForSelector("#compose_banners .dm_mention_warning", {hidden: false});
+    await common.assert_compose_box_content(page, "@_**Cordelia, Lear's daughter** ");
+    await page.click("#compose_close");
+}
```

Comment:
I think it's better to skip adding a puppeteer tests for this; they're expensive seem overkill for a banner.

Reply:
This has been removed